server/commands/battle.py

```diff
@@ -856,6 +868,146 @@ def send_info_guild(client):
+        return
+    if args[1] not in client.area.battle_items:
+        client.send_ooc("Item doesn't exist!")
+        return
+    if target.battle is None:
+        client.send_ooc("Target isn't using a fighter in this moment!")
+        return
+    target.battle.bag.append(args[1])
+    client.send_ooc(f"{args[1]} has been given to {target.battle.fighter}")
+    send_stats_fighter(target)
+    target.send_ooc(f"You have obtained {args[1]}")
+
+
+def ooc_cmd_use_item(client, arg):
+    """
+    This command will let you use an item from your bag
+    Usage: /use_item <ItemName>
+    """
+    if client.battle is None:
+        client.send_ooc("You are not using fighter in this moment!")
+        return
+    if arg not in client.battle.bag:
+        client.send_ooc("Item not found in your bag")
+        return
+    item = client.area.battle_items[arg]
+    if item["stat"] == "hp":
+        if item["operation"] == "x":
+            client.battle.hp *= item["number"]
+        else:
+            client.battle.hp += item["number"]
+        if client.battle.hp > client.battle.maxhp:
+            client.battle.hp = client.battle.maxhp
+        if client.battle.hp <= 0:
+            client.battle.hp = 1
+    if item["stat"] == "mana":
+        if item["operation"] == "x":
+            client.battle.mana *= item["number"]
+        else:
+            client.battle.mana += item["number"]
+    if item["stat"] == "atk":
+        if item["operation"] == "x":
+            client.battle.atk *= item["number"]
+        else:
+            client.battle.atk += item["number"]
+    if item["stat"] == "def":
+        if item["operation"] == "x":
+            client.battle.defe *= item["number"]
+        else:
+            client.battle.defe += item["number"]
+    if item["stat"] == "spa":
+        if item["operation"] == "x":
+            client.battle.spa *= item["number"]
+        else:
+            client.battle.spa += item["number"]
+    if item["stat"] == "spd":
+        if item["operation"] == "x":
+            client.battle.spd *= item["number"]
+        else:
+            client.battle.spd += item["number"]
+    if item["stat"] == "spe":
+        if item["operation"] == "x":
+            client.battle.spe *= item["number"]
+        else:
+            client.battle.spe += item["number"]
+
+    battle_send_ic(client, msg=f"~{client.battle.fighter}~ uses {arg} from the bag")
+
+    if (item["operation"] == "+" and item["number"] < 0) or (
+        item["operation"] == "x" and item["number"] < 1
+    ):
+        battle_send_ic(
+            client,
+            msg=f"~{client.battle.fighter}~'s {item['stat']} goes down",
+            effect="statup",
+            shake=1,
+        )
+    else:
+        battle_send_ic(
+            client,
+            msg=f"~{client.battle.fighter}~'s {item['stat']} goes up",
+            effect="statup",
+        )
+    client.battle.bag.remove(arg)
+    send_stats_fighter(client)
```

Comment:
Is there no way to set for how long is a stat modified? Status effect system also seems to be hard-coded to moves only.